### ISSUE TITLE
Adding widgets to the journal

### ISSUE THREAD

**Issue by ambiyu** [UOA-SE701-Group3-2021/3Lancers#135]
**Is your feature request related to a problem? Please describe.**
Currently, there is no way to add widgets to the journal.

**Describe the solution you'd like**
There should be a way to drop and drop to add new widgets to the journal by using the widget drawer (#93)

**Describe alternatives you've considered**
Can have a menu to select the widget and add. But drag and drop from the widget drawer is the most efficient as you can just drop to the position you like.

**Additional context**
Blocked by #93. Depends on the journal view of all the widgets (#50, #61, etc)

**Comment by ambiyu** [comment]
This issue is a duplicate of #65
